### ISSUE TITLE
Site does not meet all accessibility guidelines

### ISSUE THREAD

**Issue by hadiahmed098** [PurdueIEEE/IEEE-Website#31]
Per this [accessibility site](https://www.a11yproject.com/checklist/), Purdue IEEE does not meet some of the WCAG recommendations. While not required, we should be making sure our site is usable for all.
